go.mod

```diff
@@ -16,8 +16,9 @@ require (
 	k8s.io/apimachinery v0.18.2
 	k8s.io/client-go v0.18.2
 	k8s.io/klog v1.0.0
-	k8s.io/utils v0.0.0-20200327001022-6496210b90e8 // indirect
-	sigs.k8s.io/controller-runtime v0.5.1-0.20200330174416-a11a908d91e0
-	sigs.k8s.io/controller-tools v0.2.9-0.20200331153640-3c5446d407dd
+	sigs.k8s.io/controller-runtime v0.6.0
+	sigs.k8s.io/controller-tools v0.3.0
 	sigs.k8s.io/yaml v1.2.0
 )
+
+replace sigs.k8s.io/cluster-api-provider-aws => github.com/openshift/cluster-api-provider-aws v0.2.1-0.20200612013036-1a97c62077cb
```

Comment:
why this?

Reply:
I think without this, go.mod goes a bit mad about the dependencies because of what is checked into MAO, it doesn't want to vendor these versions so you have to have an override

https://github.com/openshift/machine-api-operator/blob/ba983a5a8c794d819e3cd7f19afc055a52de7d9e/go.mod#L29-L31

Reply:
With latest merges it will require same replaces for azure and gcp, similar to https://github.com/openshift/cluster-api-provider-azure/pull/136/files#diff-37aff102a57d3d7b797f152915a6dc16R30-R34